fdm-core/src/organization.ts

```diff
@@ -739,14 +737,14 @@ export async function rejectInvitation(
  * @param fdm - The FDM instance providing the connection to the database.
  * @param admin_id - The ID of the user initiating the removal (must be owner or admin).
  * @param organization_id - The ID of the organization from which the user is being removed.
- * @param email - The email address of the user being removed.
+ * @param username - The username of the user being removed.
  * @throws {Error} Throws an error if any database operation fails, if the user initiating the removal does not have permissions, or if the user to be removed is not found.
```

Comment:
so you decided to use an username in the app, comparable to github? no email-adresses?

Reply:
Yes, username is for now generated based on email and random number. In the future we can make users to adapt their username